### PR TITLE
feat($docs): Sync changes in Chinese

### PR DESCRIPTION
**Pull request by Mister-Hope**
**Summary**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

**Comment by Mister-Hope** [comment]
I will work on the `local-development` page tomorrow

**Comment by ulivz** [comment]
Cool, tell me if this PR is ready.

**Comment by Mister-Hope** [comment]
I think it is ready, I will call someone to have another look later.
It;s 0:45 now 😄

**Comment by Mister-Hope** [comment]
I think it's ready :smile:

File: packages/docs/docs/zh/miscellaneous/local-development.md [hunk 1]
```diff
@@ -2,4 +2,100 @@
...
+
+VuePress正在使用包含了[Yarn 工作区](https://yarnpkg.com/zh-Hans/docs/workspaces/)和[Lerna](https://github.com/lerna/lerna)的一个组合。
+
+## 初始化程序包
+
+```bash
+ yarn bootstrap // 它将运行并安装根目录所有的程序包的子文件夹
+```
+
+`yarn bootstrap`将使用`hoist`。它对您意味着什么？
```
Comment (ulivz) [inline packages/docs/docs/zh/miscellaneous/local-development.md]: > `yarn bootstrap`将使用`hoist`

这里最好加个空格：

> `yarn bootstrap` 将使用 `hoist`

File: packages/docs/docs/zh/miscellaneous/local-development.md [hunk 2]
```diff
@@ -2,4 +2,100 @@
...
+
+通过运行以下命令来检查链接：
+
+```bash
+    ls -la node_modules/@vuepress
+```
+
+您将全部链接完成。
+
+:::warning
```
Comment (ulivz) [inline packages/docs/docs/zh/miscellaneous/local-development.md]: 这里也建议加个空格：

> ::: warning

File: packages/docs/docs/zh/miscellaneous/local-development.md [hunk 3]
```diff
@@ -2,4 +2,100 @@
...
+您应该留意一个特殊的软件包，@vuepress shared-util是由Typescript编写的。
+:::
+
+安装完所有程序后，它将运行`yarn tsc`。该命令将告诉工作区的 @vuepress/shared-utils 来编译他的js。
+
+:::warning
+从这里开始，如果您要在此程序包中进行更改，则必须  
+始终运行`yarn tsc`或在单独的shell`yarn run tsc -w`中运行。当检测到shared-utils有任何更改时，它将重新运行tsc
```
Comment (ulivz) [inline packages/docs/docs/zh/miscellaneous/local-development.md]: 同上，中文和英文之前应该有一个空格

**Comment by Mister-Hope** [comment]
🤣 突然发现你的地址杭州，那我还说什么英文
好的我改一下

**Comment by Mister-Hope** [comment]
> 基本上都是空格的问题，你修复后我再看一下~

Fixed a few days ago

**Pull request closed without merging**